app/handlers.py

```diff
@@ -56,13 +57,39 @@ def join_iac(data, expected_length=12):
         return combined
 
     @staticmethod
-    def validate_case(case_json):
-        if not case_json.get("active", False):
-            raise InactiveCaseError
+    def validate_iac_active(iac_json, case_json):
+        if not iac_json.get("active", False):
+            try:
+                if case_json['caseGroup']['caseGroupStatus'] in ['COMPLETE', 'COMPLETEDBYPHONE']:
```

Comment:
Minor (pedantic!) comment but I don't think response-ops-social-ui currently supports change of status to `COMPLETEDBYPHONE`? No harm in keeping this in for future though?